core/src/test/java/kafka/testkit/TestKitNodes.java

```diff
@@ -198,7 +177,7 @@ public BootstrapMetadata bootstrapMetadata() {
         return bootstrapMetadata;
     }
 
-    public NavigableMap<Integer, BrokerNode> brokerNodes() {
+    public Map<Integer, BrokerNode> brokerNodes() {
```

Comment:
sorry that we need to use `NavigableMap` here since we instantiate broker/controller according to id.

Reply:
please ignore this comment since we create all controllers before brokers, and hence the order of brokers is not matter.

Reply:
I found all tests  failed... Sorry for wasting your time, I'll be more careful :disappointed: